R/zzz.R

```diff
@@ -23,6 +23,11 @@
   )
 }
 
-# Use non-exported function from teal.slice.
+# Use non-exported function(s) from teal.slice.
 # This is a temporary measure and will be removed two release cycles from now (now meaning 0.13.0).
 as.teal_slices <- getFromNamespace("as.teal_slices", "teal.slice") # nolint
+# This one is here because there are numerous instances of getting all ids from teal_slices.
+# The alternative is to always call vapply(<X>, `[[`, character(1), "id").
+slices_field <- getFromNamespace("slices_field", "teal.slice") # nolint
```

Comment:
I'm against creating and using syntactiv sugar for one liners based on base function. I would import this here
```suggestion
```

Reply:
removed